.github/workflows/dockerhub_latest.yaml

```diff
@@ -1,6 +1,7 @@
-name: Publish Docker image
+---
+name: Publish latest Docker image
 
-on:
+"on":
```

Comment:
Why do you quote this? It doesn't seem to be needed in other workflows.

Reply:
Well, technically it is not needed. The YAML 1.1 spec allows to use `on` as a boolean value (representing `true`). To make 100% sure YAML interpreters do not treat it as a string instead, I quoted it.

I can remove it, if you don't like it. GitHub interprets it as intended anyways.